drivers/CAN.h

```diff
@@ -94,11 +94,15 @@ class CAN : private NonCopyable<CAN> {
      * @code
      * #include "mbed.h"
      *
+     * #if defined (DEVICE_CAN) || defined(DOXYGEN_ONLY)
```

Comment:
Is this a good example? I would leave this defined from the example - if CAN not supported, an error comes from CAN header file

Reply:
Hmm, right now the CAN header file won't give an error telling you that CAN is not supported. May be, we should add that there and skip it in the example all together?